pipeline/frontend/yaml/compiler/compiler.go

```diff
@@ -108,17 +110,19 @@ func (c *Compiler) Compile(conf *yaml.Config) *backend.Config {
 	// add default clone step
 	if !c.local && len(conf.Clone.Containers) == 0 && !conf.SkipClone {
 		container := &yaml.Container{
-			Name:  "clone",
-			Image: "woodpeckerci/plugin-git:latest",
-			Vargs: map[string]interface{}{"depth": "0"},
-		}
-		// TODO: migrate to woodpeckerci/plugin-git:latest (multi arch)
-		switch c.metadata.Sys.Arch {
-		case "linux/arm":
-			container.Image = "plugins/git:linux-arm"
-		case "linux/arm64":
-			container.Image = "plugins/git:linux-arm64"
+			Name: "clone",
+			// Image:       "woodpeckerci/plugin-git:latest",
+			Image:       "woodpeckerci/plugin-git:next",
+			Vargs:       map[string]interface{}{"depth": "0"},
+			Environment: c.cloneEnv,
 		}
+		// // TODO: migrate to woodpeckerci/plugin-git:latest (multi arch)
+		// switch c.metadata.Sys.Arch {
+		// case "linux/arm":
+		// 	container.Image = "plugins/git:linux-arm"
+		// case "linux/arm64":
+		// 	container.Image = "plugins/git:linux-arm64"
+		// }
```

Comment:
Should we just remove this as the plugin is outdated (old CAs) and will be replaced with our own multiarch plugin at some point anyways?